packages/babel-plugin-proposal-optional-chaining/test/fixtures/transparent-expr-wrappers/ts-as-call-context/output.js

```diff
@@ -0,0 +1,3 @@
+var _a$b, _a;
+
+(_a$b = ((_a = a).b as any)) === null || _a$b === void 0 ? void 0 : _a$b.call(_a);
```

Comment:
@JLHwung WDYT about keeping the types? It's a nice thing to do, but I'd like to hear your opinion regarding whether or not it's worth the code complexity.

Reply:
I am good on changes here! Ideally types should only be removed via preset-flow/typescript.